### PR TITLE
[IMP] Add support for env files

### PR DESCRIPTION
**Pull request by lasley**
* Add `FILE_ENV` var in travis to allow for env files
* Support the new `FILE_ENV` in `travis_run_test`

### CODE REVIEW

**Review by tedsalmon — APPROVED** [review]
👍  LGTM!

File: travis/travis_run_test [hunk 1]
```diff
@@ -58,6 +62,10 @@ for port in ports:
 for link in links:
     cmd.append('--link=%s' % link)
 
+for env_file in file_env:
+    file_path = os.path.join(build_dir_absolute, env_file)
+    cmd.extend(('--env-file', file_path)
```
Comment (tedsalmon) [inline travis/travis_run_test]: Missing parenthesis

**Review by tedsalmon — CHANGES_REQUESTED** [review]
Nvm. Last test went red. There's a syntax error.

**Comment by lasley** [comment]
Hooray for CI!

**Review by tedsalmon — APPROVED** [review]
Alright, now that we're 🍏 , everything looks good!

**Pull request merged**